includes/global-functions.php

```diff
@@ -203,3 +240,348 @@ function distributor_get_registered_data() {
 
 	return $distributor_registered_data;
 }
+
+/**
+ * Pre-distribute callback for media data.
+ * This is the default pre-distribute callback for media data, used when the type is set to 'media' in distributor_register_data().
```

Comment:
Nit: Empty line between the short summary and the description. Applies throughout.

```suggestion
 * Pre-distribute callback for media data.
 *
 * This is the default pre-distribute callback for media data, used when the type is set to 'media' in distributor_register_data().
```